src/VisualStudio/Core/Def/Implementation/Remote/RemotableDataJsonRpcEx.cs

```diff
@@ -68,7 +68,7 @@ public async Task RequestAssetAsync(int scopeId, Checksum[] checksums, string st
             }
         }
 
-        public Task<bool> IsExperimentEnabledAsync(string experimentName, CancellationToken cancellationToken)
+        public Task<bool> IsExperimentEnabledAsync(string experimentName, CancellationToken _)
```

Comment:
This seems weird. now it's just a parameter named `_`. It can still be written and read and it's not allowed to have a second parameter named `_`. Is this just a convention we'd like people to start using for unused parameters that need to remain for override/implement/delegate conversion reasons?